websocket.go

```diff
@@ -268,6 +283,8 @@ func (c *Client) Send(dst *network.ServerIdentity, path string, buf []byte) ([]b
 	dest := destination{dst, path}
 	conn, ok := c.connections[dest]
 	if !ok {
+		// TODO we are opening a new connection for every new path?
+		// not possible to use an existing connection for the same service?
```

Comment:
Might be possible. When I started the websockets, I think I missed a lot of points and should've gone with a REST interface over http(s)...